pkg/scheduler/scheduler.go

```diff
@@ -394,6 +394,11 @@ func (sched *Scheduler) bind(assumed *v1.Pod, b *v1.Binding) error {
 // scheduleOne does the entire scheduling workflow for a single pod.  It is serialized on the scheduling algorithm's host fitting.
 func (sched *Scheduler) scheduleOne() {
 	pod := sched.config.NextPod()
+	// pod could be nil when schedulerQueue is closed
+	// add nil check to avoid triggering crash handling logic
```

Comment:
The first half of this comment is useful, but I think the second line can be removed.

Reply:
+1

Reply:
done